qemu/tests/cfg/vsock.cfg

```diff
@@ -24,3 +24,13 @@
               devno = 'fe.1.2345'
         - migration_with_vsock:
             type = migration_with_vsock
+        - iommu_enabled:
+            no RHEL.6 RHEL.7 RHEL.8.0 RHEL.8.1 RHEL.8.2 RHEL.8.3 RHEL.8.4
+            required_qemu = [5.2.0,)
+            #refer to bug 1879052,1894101
+            x86_64:
```

Comment:
```
machine_type_extra_params = "kernel-irqchip=split"
HostCpuVendor.intel:
    intel_iommu = yes
    enable_guest_iommu = yes
```
These lines may be needed.

Reply:
Thanks for the reminder @PaulYuuu 
@qcheng-redhat hello, about this test, do we need to add the following parameters on x86?

```
machine_type_extra_params = "kernel-irqchip=split"
HostCpuVendor.intel:
    intel_iommu = yes
    enable_guest_iommu = yes
virtio_dev_ats = on
```

Reply:
Hi @zhenyzha I think these parameters are for intel_iommu, as we talked in the chat.

Reply:
Ok, corrected.